### PR TITLE
Systemjs add normalize function

### PR DESCRIPTION
**Pull request by btesser**
Add missing normalize function

### CODE REVIEW

**Comment by vvakame** [comment]
@btesser I can't find `normalize` func in https://github.com/systemjs/systemjs/blob/master/docs/system-api.md

**Comment by btesser** [comment]
@vvakame 
So it may be undocumented, but it allows for providing name normalization.  For example treat `foo` as `foo.ts` instead of `foo.js`
Here is where the default method exists in the code:
https://github.com/systemjs/systemjs/blob/master/dist/system.src.js#L881

and here's an issue in their repo that references it
https://github.com/systemjs/systemjs/issues/194

File: systemjs/systemjs.d.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ interface System {
   defined: any;
   amdDefine: () => void;
   amdRequire: () => void;
+  normalize(name: string, parentName: string): string;
```
Comment (vvakame) [inline systemjs/systemjs.d.ts]: I think normalize has a 3 arguments.  `name, referrerName, referrerAddress`. not 2.

**Comment by btesser** [comment]
Good catch.  I'll add that

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**